redash/models/__init__.py

```diff
@@ -262,6 +262,10 @@ def to_dict(self):
             'retrieved_at': self.retrieved_at
         }
 
+    @classmethod
+    def get_by_id(cls, _id):
```

Comment:
@arikfr how bad is this? What was the motivation behind adding an `org_id` filter on these? The way I understand it, query results identifiers are unique.

Reply:
It's an extra safety feature to enforce security in a multi tenant context. Even if someone managed to workaround our other safe guards, he will not be able to load a query from a different account. 

TL;DR very bad :)

Reply:
9cfadd8 😖